docs/Hosting_Graphspace.md

```diff
@@ -0,0 +1,13 @@
+# Hosting GraphSpace
+
+## How to deploy the latest Github code changes on GraphSpace server? ##
+
+Here are the steps for deploying GraphSpace. It is recommended to download the [latest release](https://github.com/Murali-group/GraphSpace/releases/latest). Otherwise you can download the latest code directly from the [master branch](https://github.com/Murali-group/GraphSpace).

```

Comment:
1. Add "on a server." at the end of the first sentence. 
2. Do not use passive voice. Say "We recommend that you download the latest release. Alternately, you can download it from the ..."